docs/plugins/npe2_getting_started.md

```diff
@@ -0,0 +1,286 @@
+(npe2-getting-started)=
+
+# npe2 getting started guide
+
+This guide will walk you through the steps to write an `npe2`-style napari
+plugin from scratch.
+
+At the end of this guide, you will have a working plugin that can be installed
+with `pip` and autodetected by napari.
+
+## Overview
+
+Plugins are just python packages. They include contributions that napari
```

Comment:
I would reword to:

Plugins are just python packages with a specific napari 'manifest' file that lists 'contributions' for napari to use when performing tasks.

Reply:
reworded